### PR TITLE
fix: 🐛 apollo pending error

### PR DESCRIPTION
**Pull request by gesseekur**
### Bug

* \[x] Updated CHANGELOG.md
* \[x] Add PR label

### CODE REVIEW

**Review by gamalielhere — APPROVED** [review]

File: src/core/mixins/handlerWallet.mixin.js [hunk 1]
```diff
@@ -19,6 +19,9 @@ export default {
     getLatestPrices: {
       query: getLatestPrices,
       pollInterval: 600000,
+      skip() {
+        return !this.isEthNetwork;
```
Comment (kvhnuke) [inline src/core/mixins/handlerWallet.mixin.js:23]: This is needed, no matter the network is, this is how we get the prices for the tokens so don't skip

**Review by kvhnuke — CHANGES_REQUESTED** [review]

File: src/views/TheWalletView.vue [hunk 2]
```diff
@@ -40,7 +40,7 @@ export default {
   computed: {
     ...mapState('wallet', ['address', 'web3', 'identifier']),
     ...mapState('global', ['online', 'gasPriceType', 'baseGasPrice']),
-    ...mapGetters('global', ['network']),
+    ...mapGetters('global', ['network', 'isEthNetwork']),
```
Comment (kvhnuke) [inline src/views/TheWalletView.vue:43]: I don't think this is needed either

**Review by kvhnuke — CHANGES_REQUESTED** [review]

File: src/core/mixins/handlerWallet.mixin.js (hunk 1 above)
Comment (gesseekur) [reply src/core/mixins/handlerWallet.mixin.js]: removed

File: src/views/TheWalletView.vue (hunk 2 above)
Comment (gesseekur) [reply src/views/TheWalletView.vue]: removed

**Review by kvhnuke — APPROVED** [review]

**Review by andrewdahn — APPROVED** [review]

**Pull request merged**